### PR TITLE
Aria hide old blocks. Fix zoom glitch.

### PR DESCRIPTION
**Pull request by stephanfowler**
- Hides previous update (hidden by animation) from screen readers.
- Reduces risk of visible overflow text at certain zoom levels:

liveblog

### CODE REVIEW

**Comment by paperboyo** [comment]
Thank you, @stephanfowler!

File: static/src/javascripts/projects/facia/modules/ui/live-blog-updates.js [hunk 1]
```diff
@@ -61,7 +59,7 @@ define([
         }
 
         return template(blockTemplate, {
-            classes: block.isNew ? newBlockClassName : oldBlockClassName,
+            ariaHidden: !block.isNew,
```
Comment (sndrs) [inline static/src/javascripts/projects/facia/modules/ui/live-blog-updates.js]: :heart:

**Comment by sndrs** [comment]
:+1:

**Comment by stephanfowler** [comment]
Will repoen, after refactoring.

**Pull request closed without merging**